core-db/src/main/java/io/novafoundation/nova/core_db/migrations/50_51_AddTransactionVersionToRuntime.kt

```diff
@@ -0,0 +1,11 @@
+package io.novafoundation.nova.core_db.migrations
+
+import androidx.room.migration.Migration
+import androidx.sqlite.db.SupportSQLiteDatabase
+
+val AddTransactionVersionToRuntime_50_51 = object : Migration(50, 51) {
+
+    override fun migrate(database: SupportSQLiteDatabase) {
+        database.execSQL("ALTER TABLE chain_runtimes ADD COLUMN transactionVersion INTEGER")
```

Comment:
Have you tested this migration? I have concerns it wont work without explicitly specifying `DEFAULT NULL`

Reply:
Yes the migration is successful but I checked why: It seems that SQL use default value here and for integer it's 0, So it's good you noticed this issue. I'll fix it